backend/src/Installer.test.ts

```diff
@@ -9,6 +9,12 @@ import fs from 'fs'
 describe('backend/Installer', () => {
   describe('check', () => {
     const version = '0.37.6'
+    const versionMuxer = '0.37.6'
+    const versionDemuxer = '0.37.6'
+    const versionConnectd = '0.37.6'
+    const baseUrl = 'https://downloads.remote.it/cli/v'
+    const cliUrl = 'https://downloads.remote.it/cli/v'
+    const connectdUrl= 'https://downloads.remote.it/cli/v'
```

Comment:
Instead of handling each of the four assets differently lets just put them in a `resources` or `files` array and then loop over them. Since there is more than a version number and name (and might have more info) it could look something like this?:
```
const resources = [{
  name: 'remoteit',
  version: '6.1.14'
}, {
  name: 'connectd',
  version: '4.9'
} , {
  ...
}]
```
Actually the `installer.ts` file was designed to handle a single file install, and the `binaryInstaller.ts` was designed to iterate over instances of an installer to install each file. We could still do something like that.